src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1395,11 +1395,12 @@ def create_service_principal(cli_ctx, identifier, resolve_app=True, rbac_client=
     return rbac_client.service_principals.create(ServicePrincipalCreateParameters(app_id=app_id, account_enabled=True))
 
 
-def create_role_assignment(cli_ctx, role, assignee, resource_group_name=None, scope=None):
-    return _create_role_assignment(cli_ctx, role, assignee, resource_group_name, scope)
+def create_role_assignment(cli_ctx, role, assignee, is_service_principal, resource_group_name=None, scope=None):
+    return _create_role_assignment(cli_ctx, role, assignee, is_service_principal, resource_group_name, scope)
 
 
-def _create_role_assignment(cli_ctx, role, assignee, resource_group_name=None, scope=None, resolve_assignee=True):
+def _create_role_assignment(cli_ctx, role, assignee, is_service_principal,
```

Comment:
Maybe `is_service_principal` is a little bit duplicated? To me, it does very similar thing as `resolve_assignee`. `_resolve_object_id` can be skipped by setting `resolve_assignee` to `False`.

Reply:
Yes. That makes sense. I will get rid of this. I can just use the resolve_assignee.